### DISCUSSION TITLE
Time series: add hour or gradient/thresholds

### DISCUSSION THREAD

**Discussion by flopp999** [grafana/grafana#38223] [Dashboards, Panels & Visualizations]
I know it is Beta
I am using Grafana 7.4.1
InfluxDB as database
running on Debian 10

**What would you like to be added**:
bars colors to have "color scheme": thresholds
or possible to have two colors, e.g. green as bottom color and red as top color and gradient between them

**Why is this needed**:
I want Bars where X axis is Time, which is possible to select in Graph Panel
See image, I want hour to be present below each bar
![image](https://user-images.githubusercontent.com/21694965/108222147-cdcfff80-7138-11eb-82dc-37b9a48e9cbc.png)

![image](https://user-images.githubusercontent.com/21694965/108222236-e04a3900-7138-11eb-8ae8-a24cf1856bd4.png)

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
